packages/server/src/modules/common/i18n.js

```diff
@@ -0,0 +1,14 @@
+import i18n from 'i18n';
+
+import settings from '../../../../../settings';
+
+export default (language = i18n.getLocale(), moduleName, messageKey) => {
```

Comment:
Please remove `language` parameter it is not needed and it adds redundant work for the developer